Status[] behaviours
Status[] goals
float32  activationThreshold
float32  activationThresholdDecay